pkg/envoy/lds/inmesh_test.go

```diff
@@ -285,6 +287,35 @@ func TestBuildInboundHTTPFilterChain(t *testing.T) {
 			expectedFilterNames: []string{envoy.L4GlobalRateLimitFilterName, envoy.HTTPConnectionManagerFilterName},
 			expectError:         false,
 		},
+		{
+			name:           "inbound HTTP filter chain with global HTTP rate limiting enabled",
+			permissiveMode: true,
```

Comment:
do we have checks for `permissiveMode: false`? does this matter?

Reply:
It doesn't matter because rate limit policies apply regardless.

Reply:
No it shouldn't. Could you share the code path you are referring to?

Reply:
Ah you're right; I misremembered the code in inbound_traffic_policies.go. No objections here